composer.json

```diff
@@ -1,6 +1,25 @@
 {
+	"name": "phpbb-de/phpbb-translation",
+	"description": "German Translation of phpBB",
+	"type": "phpbb-language",
+	"homepage": "https://github.com/phpbb-de/phpbb-translation",
+	"version": "3.1.11-dev",
```

Comment:
Ich dachte wir schreiben dort nur bei einem release-tag die richtige Version rein, zu der es gehört z.B. phpBB 3.1.11 und lassen in dem branch, wo ständig was gepatcht wird, einen "-dev" Marker. Spricht da was gegen?